docs/apps/guacamole.md

```diff
@@ -1 +1,64 @@
-Placeholder page
+<h1>This guide will help you follow Gilbn's tutorial to protect your Guacamole install with F2B.</h1>
```

Comment:
Use Markdown headings
`# heading 1`

Reply:
Should be corrected now.

Reply:
@angrycuban13 You can just copy from techram and rewrite the rest of the guide to match the onzu log file and regex ect. As my guide refers to catalina.out and that might confuse some people.